uvm/js/common/util/Util.js

```diff
@@ -568,6 +568,14 @@ Ext.define('Ung.util.Util', {
         return data;
     },
 
+    getV4NetmaskMap: function() {
+        var map = {}, data = this.getV4NetmaskList();
+        data.forEach(function(element) {
+            map[element[0]] = element[1].split('-')[1].trim();
```

Comment:
Would `.split('-')[1]` ever throw ArrayIndexOutOfBoundIndex ?

Reply:
The data array returned from `this.getV4NetmaskList()` has all predefined Netmask values in format `[32, '/32 - 255.255.255.255']`
So it won't throw any exception.